pkg/k8s/client.go

```diff
@@ -76,8 +80,9 @@ func (c *KubeClient) GetPodName(podip string) (podns, poddname string, err error
 }
 
 func (c *KubeClient) getPodListWithTries(podip string, tries int) (*v1.PodList, error) {
+	phaseFilter := ",status.phase!=" + strings.Join(IGNORED_POD_PHASE_STATUSES, ",status.phase!=")
```

Comment:
Is there a way we can do this without recomputing the filter every time ?

Reply:
I’m no golang expert but I think there are two possible solutions here:
1. Write the filter string in a hard-coded way
2. Create a package wide variable For the filter and create a getOrCreateFilter method that will initialize the variable once and return its value

WDYT?